src/components/Media/Media.tsx

```diff
@@ -35,40 +29,27 @@ const Media: React.FC<MediaProps> = ({ items, activeItem }) => {
   return (
     <MediaWrapper>
       {items.map((item: object, key: number) => {
```

Comment:
I'd kind of like to break the code logic out of what's being `return()d`.  It's easier for my brain to read.  What if after `handleChange` declaration, you created a new variable like `const displayItems = items.map(......)`;  and then the return below looks something like:

```
return (
  <MediaWrapper>
    {displayItems.map(item => <MediaItem ... />)}
  </MediaWrapper>
)

Something like that?
```